Android/ConstraintLayout/source/ConstraintLayout/ConstraintLayout.csproj

```diff
@@ -68,5 +114,26 @@
       <Name>ConstraintLayoutSolver</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Android.Support.v7.RecyclerView">
+      <Version>28.0.0-preview7</Version>
+    </PackageReference>
+    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat">
+      <Version>28.0.0-preview7</Version>
+    </PackageReference>
+  </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.Bindings.targets" />
+  <Import Project="..\..\samples\packages\Xamarin.Android.Support.Annotations.28.0.0-preview7\build\monoandroid81\Xamarin.Android.Support.Annotations.targets" Condition="Exists('..\..\samples\packages\Xamarin.Android.Support.Annotations.28.0.0-preview7\build\monoandroid81\Xamarin.Android.Support.Annotations.targets')" />

```

Comment:
All of these .targets Import's need to get deleted too.  Again, PackageReference implicitly adds them...